app/models/katello/system.rb

```diff
@@ -69,16 +62,18 @@ class System < Katello::Model
     before_update :update_foreman_host, :if => proc { |r| r.environment_id_changed? || r.content_view_id_changed? }
 
     scope :in_environment, ->(env) { where('environment_id = ?', env) unless env.nil? }
-    scope :completer_scope, ->(options) { readable(options[:organization_id]) }
```

Comment:
I'm not able to reproduce, I only see search filters on this page that seem to be working correctly. Is this right? http://www.clipular.com/c/5617585759453184.png?k=866e_fm6XzsotQHQQq7kvqzr9pw